opendebates/tests/test_vote.py

```diff
@@ -294,6 +306,53 @@ def test_vote_user_disabled_captcha(self):
         refetched_sub = Submission.objects.get(pk=self.submission.pk)
         # ... and in DB
         self.assertEqual(self.votes + 1, refetched_sub.votes)
+        self.assertEqual(self.current_votes + 1, refetched_sub.current_votes)
+
+    def test_vote_after_previous_debate(self):
+        "Votes after the previous debate get tracked separately."
+        mode, _ = SiteMode.objects.get_or_create()
+        mode.previous_debate_time = timezone.make_aware(
+            datetime.datetime(2016, 1, 1, 12)
```

Comment:
Same comment as above - could we derive this from timezone.now() to be more explicit about what matters to the test?